akvo/rsr/tests/rest/test_indicator_period_data.py

```diff
@@ -1669,3 +1670,178 @@ def test_query_audit_trail(self):
             }
         ]
         self.assertEqual(expected, detail_response.data['audit_trail'])
+
+
+class IndicatorPeriodDataCollaborateDraftTestCase(BaseTestCase):
+    """
+    Tests the collaboration on indicator period update feature.
+    Currenly only Nuffic requested this feature.
+    """
+
+    def create_user_org(self, email, org, role='Enumerator'):
+        user = self.create_user(email, 'password')
+        self.make_employment(user, org, role)
+        return user
+
+    def create_users(self, usernames, org):
+        return (self.create_user_org(f'{u}@acme.org', org, 'Enumerators') for u in usernames)
+
+    def make_project_with_one_period(self, org):
+        project = ProjectFixtureBuilder()\
+            .with_partner(org, Partnership.IATI_REPORTING_ORGANISATION)\
+            .with_results([{
+                'title': 'Result  #1',
+                'indicators': [{
+                    'title': 'Indicator #1',
+                    'periods': [{
+                        'period_start': date(2010, 1, 1),
+                        'period_end': date(2010, 12, 31),
+                    }]
+                }]
+            }]).build()
+        period = project.get_period(period_start=date(2010, 1, 1))
+        return project, period
+
+    def enable_collab_feature_for(self, project):
+        return override_settings(NUFFIC_ROOT_PROJECT=project.object.id)
+
+    def test_get_results_framework__project_without_collab_feature__should_not_be_able_to_see_other_user_draft_update(self):
+        org = self.create_organisation('Acme')
+        user1, user2 = self.create_users(['user1', 'user2'], org)
+        project, period = self.make_project_with_one_period(org)
+        period.add_update(user2, value=1, status='D')
```

Comment:
Could you use the constants e.g `IndicatorPeriodData.STATUS_DRAFT_CODE`  instead of literals e.g 'D'? It isn't immediately obvious what it means.